status-im/status-react
# chore: remove mock data from wallet and feature flag incomplete features

fixes: https://github.com/status-im/status-mobile/issues/18378

This pr removes mock data for the following features

Removed mock data from Collectibles Overview Page
- About Tab
- Activity Tab
<img src="https://github.com/status-im/status-mobile/assets/22799766/4b9f0366-0616-4385-b236-0b99d50c9dff" width="200px" />
<img src="https://github.com/status-im/status-mobile/assets/22799766/f95b5898-2f2d-4c23-84f1-d0ac2a9f61ff" width="200px" />

Added Feature flag for Bridge page and for Edit Derivation Path
https://github.com/status-im/status-mobile/assets/22799766/eccd0fe1-8a8c-421b-a8a4-529cd0e27393

≈i.e on this page:

<img width="226" alt="Screenshot 2024-01-18 at 23 39 26" src="https://github.com/status-im/status-mobile/assets/22799766/88310f56-4f6a-45f7-8562-688b98b20d96">


Note: I will create features to correctly restore all of these features with real data and with that I will link this pr so that the respective data structures are not lost 👍 

Happy to rework this approach if there is a preferable alternative.

To test:

View a collectible page - you will see the activity is now not displaying
you will also see the socials links are now not showing - these were on the about tab of the collectibles view

Also on the Account input there is no suggested addresses showing now - these were mock previously.
i.e go to account page - click send button - start inputting an address 
- no suggestions should show

**Added Feature flag for Bridge page** 
- Go to Accounts View
- Click Bridge Button
- Blocked by feature flag (alert shows)

**Added Feature flag for Edit Derivation Path**
- Go to Create Account Page
- hit edit derivation path
- Blocked by feature flag (alert shows)
